.circleci/common.sh

```diff
@@ -104,6 +104,7 @@ function build_torch_xla() {
   XLA_DIR=$1
   pushd "$XLA_DIR"
   python setup.py install
+  pip install plugins/cuda -v
```

Comment:
will this command both build and install the cuda plugin?

Reply:
Yes